### ISSUE TITLE
Error: author field is required. 

### ISSUE THREAD

**Issue by ayusufAB** [circleci/circleci-docs#5593]
# Description and Steps taken so far
Interactive Orb Development Kit setup stops after i enter my Github repo URL and hit enter. The following error is displayed:

circleci orb init /Users/ahmedyusuf/secrets
Note: This command is in preview. Please report any bugs! https://github.com/CircleCI-Public/circleci-cli/issues/new/choose
? Would you like to perform an automated setup of this orb? Yes, walk me through the process.
Downloading Orb Project Template into /Users/ahmedyusuf/secrets
A few questions to get you up and running.
? Are you using GitHub or Bitbucket? GitHub
? Enter your github username or organization 1Password
? Enter the namespace to use for this orb onepassword
Saving namespace onepassword as default
? What categories will this orb belong to? 
? Automatically set up a publishing context for your orb? Yes, set up a publishing context with my API key.
`orb-publishing` context already exists, continuing on
? Would you like to set up your git project? Yes
? Enter your primary git branch. main
? Enter the remote git repository https://github.com/1Password/onepassword-secrets-orb.git
Thank you! Setting up your orb...
Error: author field is required
`

This is the first time authoring CircleCI orbs so haven't got this to work so far.

Running the latest CLI version 

Operating system:
MacOS Big Sur 11.4

**Comment by thomasr8** [comment]
Hi @ayusufAB : Thank you for filing this issue. If I may ask, are you currently using the personal access token of the owner of the organization? According to a developer I chatted with about this issue, he mentioned that you most likely are using a personal access token that does not have the correct permissions set.

I am copying our developer @KyleTryon to this, and he also recommended that you send issues to the circleci-cli repo.

Please let me know if you are still having issues, and I will try and help as much as I can.

Thank You.

**Comment by ayusufAB** [comment]
Hey @thomasr8, Thank you for responding to this. Yes, Im currently using a personal access token as one of the owners of the organization.

A few days ago I removed CircleCi as a Authorised OAuth app after we created the namespace the first time around. 

Once we decided to go ahead with the completing the setup,  I went ahead and re-setup permissions again, refreshed my personal access token and an orb-publishing context.  
Note: I only ran the Interactive Orb Development Kit setup after re-setting up permissions again, didnt try it the first time around.

Could removing CircleCi permissions the first time around broke something?
